entrypoint.sh

```diff
@@ -150,5 +157,12 @@ set_download_url
 check_version_exists
 download_aws_cli
 install_aws_cli
+if [[ $_LIGHTSAIL_INSTALL = "true" ]]; then
+    if [[ $_AWS_CLI_VERSION =~ ^2.*$ ]]; then
+        install_lightsailctl
+    else
+        msg_error "Cannot install Lightsail plugin with CLI 1.x"
+    fi
+fi
 cleanup
 test_aws_cli
```

Comment:
Please add a test to make sure that lightsail is installed, for example:
```bash
test_lightsailctl(){
  local installed_lightsail
  if [[ $_LIGHTSAIL_INSTALL  = "true" ]]; then
    installed_lightsail=$(lightsailctl 2>&1 1>/dev/null)
    if [[ $installed_lightsail =~ "it is meant to be invoked by AWS CLI" ]]; then
      log_msg "Lightsail was installed successfully"
    else
      error_msg "Failed to install lightsailctl"
    fi
  fi
}
```

Invoke this test after `test_aws_cli`